src/components/TripsList.jsx

```diff
@@ -22,8 +22,8 @@ export const TripsList = ({ account }) => {
     tripsQuery.options
   )
 
-  const isLoading =
-    isQueryLoading(tripsQueryResult) && !hasQueryBeenLoaded(tripsQueryResult)
+  const isLoading = isQueryLoading(tripsQueryResult)
```

Comment:
Do we have a fetchMore() button in the TripsList? If yes, we should keep the `hasQueryBeenLoaded` check